ajax/libs/print-js/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "print-js",
+  "filename": "print.min.js",
+  "homepage": "http://printjs.crabbly.com",
+  "author": "Rodrigo Vieira <rodrigo@crabbly.com>",
+  "description": "A tiny javascript library to help printing from the web.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/crabbly/Print.js.git"
+  },
+  "keywords": [
+    "javascript",
+    "print.js",
+    "printjs"
+  ],
+  "npmName": "print-js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "src",
+      "files": [
+        "css/print*"
```

Comment:
No files appear to match this in the package:

```
node_modules/print-js
├── LICENSE
├── README.md
├── dist
│   ├── print.css
│   ├── print.js
│   └── print.map
├── package.json
└── src
    ├── index.d.ts
    ├── index.js
    ├── js
    │   ├── browser.js
    │   ├── functions.js
    │   ├── html.js
    │   ├── image.js
    │   ├── init.js
    │   ├── json.js
    │   ├── modal.js
    │   ├── pdf.js
    │   ├── print.js
    │   └── raw-html.js
    └── sass
        ├── index.scss
        ├── modules
        │   └── _colors.scss
        └── partials
            ├── _modal.scss
            └── _spinner.scss
```

Reply:
For versions below 1.0.46 this one will get css file inside css folder